src/ovp_pipeline/unified_pipeline_enhanced.py

```
@@ -2890,26 +2909,26 @@ def main():
             print(f"  API Key: {key}...")
             print(f"  API Base: {os.environ.get('AUTO_VAULT_API_BASE', 'N/A')}")
         else:
-            print("\n✗ 环境未就绪，请运行: ovp --init")
+            print("\n✗ 环境未就绪，请确认 --vault-dir/VAULT_DIR 指向 vault，并检查 .env")
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)